package.json

```diff
@@ -18,7 +18,8 @@
     "lintjs": "eslint . --no-eslintrc --config .eslintrcforjs.js --ext .js",
     "lint": "npm run lintts && npm run lintjs",
     "format": "clang-format --glob=\"{{src,test}/**/*.ts,{src,test}/**/*.js}\" --style=file -i",
-    "test": "cross-env NODE_ENV=test mocha --require ./test/node_setup.js --exit --recursive"
+    "test": "cross-env NODE_ENV=test mocha --require ./test/node_setup.js --exit test/*/*.js",
+    "cts": "cross-env NODE_ENV=test NODE_OPTIONS='--max-old-space-size=8192' mocha --require ./test/node_setup.js --exit test/cts/from_nnapi/tests/cts.js"
```

Comment:
Could `test` include the cts test cases?

Reply:
@huningxin I didn't involve cts test cases into test since it spent about 40 minutes to run cts test cases in node.js, and I think tests ran in CI should be not hard, CI should show checking result quickly. What's your opinion? thanks.

Reply:
I investigated this issue. The root cause is the babel-register and babel-poyfill processing for the big cts.js file. My solution is to enable the native es6 module support so we don't need the babel anymore. I put together a PR https://github.com/BruceDai/webnn-polyfill/pull/2 to your repo.

To my test, with this change, the cts test can finish within 1 second. Please have a try.

Reply:
@huningxin I verified that cts test could be ran quickly in 1 second. Thanks for fixing it.

Reply:
Great to know!

Reply:
And I am fine to leave the cts test standalone in current stage.